be/src/olap/task/engine_alter_tablet_task.cpp

```diff
@@ -25,7 +25,14 @@ namespace doris {
 using std::to_string;
 
 EngineAlterTabletTask::EngineAlterTabletTask(const TAlterTabletReqV2& request)
-        : _alter_tablet_req(request) {}
+        : _alter_tablet_req(request) {
+    _mem_tracker = MemTracker::create_tracker(
+            config::memory_limitation_per_thread_for_schema_change * 1024 * 1024 * 1024,
```

Comment:
we should put unit in config name, e.g. memory_limitation_per_thread_for_schema_change_gb

Reply:
done, thats right